index.js

```diff
@@ -1,1144 +0,0 @@
-    console.log(green.dialogueOne);
-
-    if (!player.hasSpokenTo.includes("green")) {
-      player.hasSpokenTo.push("green");
-    }
-    playKitchen();
-  } else if (inputArr.includes("take") && inputArr.includes("pan")) {
-    takeItem(kitchen,"Frying Pan");
-    playKitchen();
-  } else if (inputArr.includes("take") && inputArr.includes("notebook")) {
-    if (player.hasMoved === true) {
-      takeItem(kitchen, "Notebook");
-    } else {
-      console.log("\nThere is no notebook to be seen.");
-    }
-    playKitchen();
-  } else if (inputArr.includes("examine") && inputArr.includes("notebook")) {
-    if (player.inventory.includes("Notebook")) {
-      console.log(notebook.desc);
-      playKitchen();
-    } else {
-      console.log("\nYou are not in possesion of a notebook.");
-      playKitchen();
-    }
-  } else if (inputArr.includes("take") && inputArr.includes("key")) {
-    takeItem(kitchen, "Key");
-    playKitchen();
-  } else if (inputArr.includes("use") && inputArr.includes("stove")) {
-    stove.action();
-    playKitchen();
-  } else if (inputArr.includes("drop") && inputArr.includes("pan")) {
-    dropItem(kitchen, "Frying Pan");
-    playKitchen();
-  } else if (inputArr.includes("drop") && inputArr.includes("notebook")) {
-    dropItem(kitchen, "Notebook");
-    playKitchen();
-  } else if (inputArr.includes("drop") && inputArr.includes("key")) {
-    dropItem(kitchen, "Key");
-    playKitchen();
-  } else if (inputArr.includes("go") && inputArr.includes("dining")) {
-    player.location = "The Dining Room";
-    console.log(diningMessage);
-    playDining();
-  } else if (inputArr.includes("go") && inputArr.includes("conservatory")) {
-    player.location = "The Conservatory";
-    console.log(conservatoryMessage);
-    playConservatory();
-  } else if (inputArr.includes("go") && inputArr.includes("library")) {
-    player.location = "The Library";
-    console.log(libraryMessage);
-    playLibrary();
-  } else if (inputArr.includes("go") && inputArr.includes("study")) {
-    player.location = "The Study";
-    console.log(studyMessage);
-    playStudy();
-  } else if (inputArr.includes("go") && inputArr.includes("lounge")) {
-    player.location = "The Lounge";
-    console.log(loungeMessage);
-    playLounge();
-  } else if (inputArr.includes("go") && inputArr.includes("billiard")) {
-    player.location = "The Billiard Room";
-    console.log(billiardMessage);
-    playBilliardRoom();
-  } else if (inputArr.includes("go") && inputArr.includes("ballroom")) {
-    player.location = "The Ballroom";
-    console.log(ballroomMessage);
-    playBallroom();
-  } else if (inputArr.includes("look") && inputArr.includes("around")) {
-    console.log(kitchen.desc + "\n\nItems include: " + kitchen.inventory);
-    playKitchen();
-  } else {
-    console.log(
-      `\nI don't understand what you want and/or you can't do that in this room...`
-    );
-    playKitchen();
-  }
-}
```

Comment:
This function is extremely similar to each play*Room* function. this could have been cleaned up and removed quite a bit of code if you created a single function with the argument being the room they are in or the room they are trying to go to.

Reply:
I completely agree. Originally we thought we'd be saving code by not having all commands available in each room but that quickly realized that the bulk of commands available in each room are redundant. I will definitely be compiling the command logic into a single play() function and just use the player.location argument.

Reply:
Fixed